nicegui/elements/json_editor.py

```diff
@@ -42,3 +43,7 @@ def properties(self) -> Dict:
     def update(self) -> None:
         super().update()
         self.run_method('update_editor')
+
+    def call_editor_method(self, name: str, arg: Optional[str] = None, timeout: float = 1,
```

Comment:
I would suggest to name the function `run_api_method`. We already provide a `run_method` function from the base class `Element` and the naming should be consistent. In AgGrid we have named it `call_api_method` which we should also change in the next major release.

Reply:
done